### ISSUE TITLE
[Footer]: QA: The tabbable element's role 'none' is not a widget role

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#8999]
### Description

Element code:
<dds-masthead-logo
    tabindex="0"
    slot="brand"
    size=""
    data-autoid="dds--masthead-logo">

Issue : The tabbable element's role 'none' is not a widget role

Expected: If users should not be able to interact with this element using the keyboard, replace the non-widget role, and/or use the explicit tabindex="-1" attribute to remove the element from the keyboard focus (tab) order.
Or,if users should be able to interact with this element using the keyboard, use a valid widget role

### Component(s) impacted

Footer

### Browser

Firefox

### Carbon for IBM.com version

v1.35.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
